src/bun.js/webcore.zig

```diff
@@ -479,7 +479,7 @@ pub const Crypto = struct {
                 if (p_int < 0 or !value.isNumber()) {
                     return throwInvalidParams(
                         globalThis,
-                        "RangeError",
+                        .range_error,
```

Comment:
this should be PascalCase IMO

.RangeError

much harder to guess as .range_error